actions/recipe_test.go

```diff
@@ -1,7 +1,7 @@
 package actions_test
 
 import (
-	"github.com/go-debos/debos/recipe"
+	"github.com/go-debos/debos/actions"
```

Comment:
This should be combined in the patch that moves things around so we don't have random commits that fail tests

Reply:
Squashed into commit moving recipe/recipe.go to actions/recipe.go